settings.local.generic.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Local Backdrop CMS configuration file.
+ */
+
+/**
+ * Simple database settings:
+ *
+ * Most sites can configure their database by entering the connection string
+ * below. If using master/slave or multiple connections, see the advanced
```

Comment:
Let's not reintroduce this terminology :) I think it's called `primary/replica` now.

Reply:
That was just copied from the original settings file. There's not even a need for the general-purpose language in this specific settings.local.php file, so I've removed it entirely and instead have comments specific to this use case.